cvpy/tests/test_biomedimage.py

```diff
@@ -333,6 +333,61 @@ def test_mask_encoded_image_decoded_mask(self):
 
         # Compare the arrays
         return np.array_equal(export_img_arr, test_arr)
+    
+    def test_morphological_gradient_3d_grayscale_image(self):
+        # Load the input image
+        input = self.s.CASTable('input')
+        self.s.image.loadimages(path='biomedimg/Prostate3T-01-0001.nii',
+                                caslib='dlib',
+                                casout=input)
+
+        input_table = ImageTable(input)
+
+        # Construct Biomed object
+        biomed = BiomedImage(cas_session=self.s)
+
+        # Compute Morphological Gradient
+        output = biomed.morphological_gradient(images=input_table, copy_vars = ['_id_'])
+
+        # Assert the output
+        self.assertTrue(output.table is not None)
```

Comment:
Do you think we should compare the output image to a benchmark image saved to fvcc?

Reply:
Yes, that would be a great idea. Because I was not sure what else I could assert here. 
So this would be done by calling matchImages. Let me try that.